src/shared/components/palette/collection/write-collection-field.component.ts

```diff
@@ -69,7 +69,7 @@ export class WriteCollectionFieldComponent extends AbstractFieldWriteComponent i
     });
   }
 
-  ngOnDestroy() {
+  public ngOnDestroy(): void {
     if (typeof this.profileSubscription !== 'undefined') {
```

Comment:
This doesn't account for it being `null` and, for the sake of safety, probably should.
```suggestion
    if (this.profileSubscription) {
```